README.md

```diff
@@ -113,244 +112,128 @@ bash test/test_cleanup.sh
-name it as you like, e.g. `my_lib` and move there.    
+Place your library file(s) here. In addition, create the sample table 
+`sample_table.csv` and fill it up. The `sample_table.csv` is a tab-separated  
+file, with one row per library, that must have the following columns:  
 
-```bash
-mkdir my_lib
-cd my_lib
-```
-Place your library file(s) here. 
-In addition, create a sample table. 
-Fill it with the correct entries. 
-You can look at the `test/test_files/sample_table.csv` 
-to get an idea of what this file must look like.
+- `sample`. This column contains the library name.  
+- `sample_file`. In this column, you must provide the path to the library file.
+The path must be relative to the working directory.  
+- `adapter`.  This field must contain the adapter sequence in capital letters.  
+- `format`. In this field you mast state the library format. It can either be 
+`fa` if providing a FASTA file or `fastq` if the library is a FASTQ file.  
+
+You can look at the `test/test_files/sample_table.csv` to know what this file 
+must look like, or use it as a template.
 
 ```bash
 touch sample_table.csv
 ```
+In this same directory, add the following required input files:  
+
+- The gzipped reference genome in FASTA format.  
+- The gzipped gene annotation file in GTF format.  
+- The unzipped microRNA annotation file in GTF format.  
+- A tab-separated mappings table between UCSC and Ensembl.
 
-Now traverse back to the previous directory.
-Here, a couple of things must be done:  
+> **Note:** We expect the genome and gene annotations to be formatted according
+> the style used by Ensembl. Other formats are very likely to lead to problems.
+> The miRNA annotation file is expected to originate from miRBase, or follow 
+> their exact layout.
 
-First of all, copy the `config_template.yaml` to this directory.
+Finally, copy the `config_template.yaml` to this directory.
 
 ```bash
 cp ..config/config_template.yaml ./config.yaml
 ```
 Then, using your editor of choice, adjust the parameters of the `config.yaml`.
 The file explains what each of the parameters means and how you can meaningfully
-fill them in.  
+fill them in. 
 
+### Running the workflow locally
 
-Accordingly to how you want
-to run the workflow you can either 
-copy the script to run it **locally** with
-
-```bash
-cp ../test/test_workflow_local.sh ./run_workflow_local.sh
-```
-or copy the script to run the workflow 
-on a **cluster via Slurm** 
-along with the `cluster.json` file with
-
-```bash
-cp ../test/test_workflow_slurm.sh ./run_workflow_slurm.sh
-cp ../test/cluster.json ./cluster.json
-```
 
-In both cases, the final 9 lines must be removed; this can be done with:
+With all the require files on the directory you can run the workflow with the
+following command:  
 
 ```bash
-head -n 9 run_workflow_*.sh
+snakemake \
+    --snakefile="../workflow/Snakefile" \
+    --cores 4  \
+    --use-singularity \
+    --singularity-args "--bind ${PWD}/../" \
```

Comment:
Why bind the directory upstream of your current working directory if you are in your repo root?
Anyway, this param probably needs an explanation, else people are bound to get into trouble.

Reply:
During the testing, the working directory is the `test/` therefore, the upstream directory is the repo root. I did not change it as I assumed that the user will create a directory to run the workflow thus, it will be the same as running it from the `test/` directory.

Should I change `${PWD}` for `src` and `../` for `dest` and give a couple of notes as well as referring the user to the documentation for more info?

Reply:
Ah yes, you are right. The best is to write scripts in a way that they can be executed from anywhere. But this is not a documentation issue, so you can open an issue for that and leave it as it is for this PR.

Reply:
I quite don't get what the issue should look like ^^'

Reply:
_Ensure that test scripts can be successfully run from any location_